2018/07.md

```diff
@@ -72,7 +72,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | timebox | topic | presenter |
     |:-------:|-------|-----------|
 
-1. Timeboxed overflow from previous meeting
+1. Overflow from previous meeting
 
     | timebox | topic | presenter |
     |:-------:|-------|-----------|
```

Comment:
Hmm I'm not sure what to do with blooks which were scheduled for 45 minutes but got 10 minutes.

I guess I'm OK in this particular instance not reserving time, but I think we should be cautious in the future of assuming that giving someone a ten-minute lightning talk is enough to remove their overflow privileges.

Reply:
I don't think overflow "privileges" are something we should be relying on generally. The idea is to try to fit everyone's topic in, and we all compromise on time to make that happen, using overflow as a last resort.